src/port/port.tcl

```diff
@@ -2171,9 +2155,40 @@ proc action_info { action portlist opts } {
                 set label "$opt: "
             }
 
+            if {$ropt in {"description" "long_description"}} {
```

Comment:
It is an artifact of the removal of `[join $portinfo($ropt)]` above, which is required because it would otherwise flatten the maintainer list. Not running the `[join ...]` for descriptions breaks ports that use newlines in their descriptions, e.g. `barcode`.